src/client/containers/DeleteBoardModal/DeleteBoardModal.style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+.close-btn-div {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 0.9em;
+  padding-top: 0.9em;
+}
+
+.greyLine {
+  height: 5px;
+  width: 86px;
+  border: 1px solid #e5e5e5;
+  background-color: rgb(228, 222, 222);
+  border-radius: 5px;
+  margin: auto;
+}
+
+.delete-board-container {
+  width: 414px;
+  height: 328px;
+  background-color: #fff;
+  border-radius: 30px 30px 0 0;
+  padding: 1em;
+  position: absolute;
+  bottom: 0;
+  z-index: 100;
+  border: 1px solid grey;
+}
+
+.circle-exclamation {
+  border: 3px solid #5d0096;
+  border-radius: 50%;
+  width: 42px;
+  height: 42px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 50;
+}
+
+.content-container {
```

Comment:
These names a too generic. What content? and what container?

Reply:
Yes it is ... I will change it